src/integrationTest/java/uk/gov/hmcts/reform/judicialapi/elinks/DeletedIntegrationTest.java

```diff
@@ -68,7 +68,7 @@ void shouldLoadDeletedApiData(TestDataArguments testDataArguments) throws Except
 
         verifyUserJudiciaryRolesData(testDataArguments.expectedRoleSize());
 
-        verifyDeletedDataLoadAudit(testDataArguments.expectedJobStatus());
+        verifyDeletedDataLoadAudit(testDataArguments.expectedJobStatus(), 4);
```

Comment:
use expectedAuditRecords from testDataArguments instead of hard coding

Reply:
done